### PR TITLE
Copy to clipboard

### PR DESCRIPTION
**Pull request by andolojm**
Add a button to copy the textarea contents to the user's clipboard

### CODE REVIEW

File: src/components/SpellBuilder/SpellBuilder.vue [hunk 1]
```diff
@@ -39,17 +39,24 @@
...
 
 <script>
 import Vue from 'vue'
+import VueClipboard from 'vue-clipboard2'
 import Grid from '@/Domain/Grid/Grid'
 import FrameCleaner from '@/Domain/Frame/FrameCleaner'
 import PlayerNode from '@/components/SpellBuilder/Nodes/PlayerNode'
 import SpellNode from '@/components/SpellBuilder/Nodes/SpellNode'
 
+Vue.use(VueClipboard)
```
Comment (laroccrj) [inline src/components/SpellBuilder/SpellBuilder.vue:58]: I believe you can add this to the components array instead of vue,use

Comment (laroccrj) [reply src/components/SpellBuilder/SpellBuilder.vue:58]: it's an object, not an arRAy, but you feel me

Comment (andolojm) [reply src/components/SpellBuilder/SpellBuilder.vue:58]: I just copied VueClipboard to the object and it doesn't work, but I'll keep looking and see if there's another way

Comment (andolojm) [reply src/components/SpellBuilder/SpellBuilder.vue:58]: I believe it is a plugin and not a component so Vue.use is the correct way to do it

https://vuejs.org/v2/guide/plugins.html

_______________

Adding it to main.js where vue is instantiated and having all the plugins there would be nice but it doesn't seem to work in the component without it there

Comment (laroccrj) [reply src/components/SpellBuilder/SpellBuilder.vue:58]: Ah alright, cool, good to know

**Pull request merged**